python_modules/dagster/dagster/_utils/aiodataloader.py

```diff
@@ -100,13 +152,17 @@ def __init__(
         self._cache = cache_map if cache_map is not None else {}
         self._queue: List[Loader] = []
 
+        super().__init__(batch_load_fn, max_batch_size)
+
     @property
     def loop(self) -> AbstractEventLoop:
         # ensure a good error is thrown if the event loop has changed since instantiation,
         # can result in cryptic errors otherwise.
 
         # it might be possible to instead update the event loop if there is no work in flight,
         # unclear if there are valid uses cases for that
+        if self._loop is None:
+            self._loop = get_event_loop()
```

Comment:
still want this change?

Reply:
it's actually somewhat necessary in order to let us unit test without setting up any async stuff in the background (otherwise get_event_loop() in the constructor will error)

Reply:
worth a comment i think, especially if the only motivation is unit test